core/domain/user_jobs_one_off_test.py

```diff
@@ -1585,6 +1581,93 @@ def test_one_setting_model_without_gae_id(self):
             migrated_setting_model.last_updated)
 
 
+class MockUserSubscriptionsModelWithFeedbackThreadIDs(
+        user_models.UserSubscriptionsModel):
+    """Mock UserSubscriptionsModel so that it allows to set
+    `feedback_thread_ids`.
+    """
+
+    feedback_thread_ids = (
+        datastore_services.StringProperty(indexed=True, repeated=True))
+
+
+class RemoveFeedbackThreadIDsOneOffJobTests(test_utils.GenericTestBase):
+    def _run_one_off_job(self):
+        """Runs the one-off MapReduce job."""
+        job_id = (
+            user_jobs_one_off.RemoveFeedbackThreadIDsOneOffJob.create_new())
+        user_jobs_one_off.RemoveFeedbackThreadIDsOneOffJob.enqueue(job_id)
+        self.assertEqual(
+            self.count_jobs_in_mapreduce_taskqueue(
+                taskqueue_services.QUEUE_NAME_ONE_OFF_JOBS), 1)
+        self.process_and_flush_pending_mapreduce_tasks()
+        stringified_output = (
+            user_jobs_one_off.RemoveFeedbackThreadIDsOneOffJob
+            .get_output(job_id))
+        eval_output = [ast.literal_eval(stringified_item) for
+                       stringified_item in stringified_output]
+        return eval_output
+
+    def test_one_subscription_model_with_feedback_thread_ids(self):
+        with self.swap(
+            user_models, 'UserSubscriptionsModel',
+            MockUserSubscriptionsModelWithFeedbackThreadIDs):
+            original_setting_model = (
+                user_models.UserSubscriptionsModel(
+                    id='id',
+                    feedback_thread_ids=[]
```

Comment:
This should fix the coverage:
```suggestion
                    feedback_thread_ids=['some_id']
```

Reply:
Thanks, I was trying to get the tests passed. Done!